geomesa-kafka/geomesa-kafka-datastore/src/test/scala/org/locationtech/geomesa/kafka/LiveKafkaConsumerFeatureSourceTest.scala

```diff
@@ -105,6 +109,60 @@ class LiveKafkaConsumerFeatureSourceTest extends Specification with HasEmbeddedK
         features.hasNext must beFalse
       }
     }
+
+    "support listeners" >> {
+      val m = AtomicLongMap.create[String]()
+
+      val id = "testlistener"
+      val numUpdates = 100
+      val maxLon = 80.0
+      var latestLon = -1.0
+
+      val sft = {
+        val sft = SimpleFeatureTypes.createType("listeners", "name:String,age:Int,dtg:Date,*geom:Point:srid=4326")
+        KafkaDataStoreHelper.createStreamingSFT(sft, zkPath)
+      }
+      val producerDS = DataStoreFinder.getDataStore(producerParams)
+      producerDS.createSchema(sft)
+
+      val listenerConsumerDS = DataStoreFinder.getDataStore(consumerParams)
+      val consumerFC = listenerConsumerDS.getFeatureSource("listeners")
+
+      val latch = new CountDownLatch(numUpdates)
+
+      val fl = new TestLambdaFeatureListener((fe: KafkaFeatureEvent) => {
```

Comment:
[style]  As a reader, it would help me if we replaced variables such as ```fl``` with something more descriptive like, ```featureListener```.  I'm happy to be overridden.